src/mono/wasm/templates/templates/browser/.template.config/template.json

```diff
@@ -10,5 +11,25 @@
   "tags": {
     "language": "C#",
     "type": "project"
+  },
+  "symbols": {
+    "kestrelHttpPortGenerated": {
+      "type": "generated",
+      "generator": "port",
+      "parameters": {
+        "low": 5000,
+        "high": 5300
+      },
+	    "replaces": "5000"
+    },
+    "kestrelHttpsPortGenerated": {
+      "type": "generated",
+      "generator": "port",
+      "parameters": {
+        "low": 7000,
```

Comment:
Maybe not relevant for this PR, but I noticed that the HTTPS didn't have a cert which my browser would trust. On the other hand any ASP.NET project has some dev cert installed.  Could we also use it ?